### DISCUSSION TITLE
Trying to use custom CSS but get "MIME type ('text/html') is not a supported stylesheet MIME type" error

### DISCUSSION THREAD

**Discussion by LukeTwomey** [giscus/giscus#877] [Q&A] [answered]
So I've got a NextJS app, to which I've added a css file into my public/styles folder.  I'm pointing to that in my Giscus settings, like so:

<img width="296" alt="image" src="https://user-images.githubusercontent.com/7040126/216023208-c534ead9-83bb-48c5-9bdb-d5478f4208b0.png">

And when the page loads, I get the following error at the console:

<img width="1226" alt="image" src="https://user-images.githubusercontent.com/7040126/216023390-2a7f391b-4d22-46d8-b691-c6cd3af891a6.png">

Am I doing this incorrectly? Is there a way to specify the MIME type in the theme?

**Comment by laymonage** [comment, marked as answer]
You cannot use a relative URL for the theme at the moment (as it will be relative to https://giscus.app), so try specifying a full URL.

**Reply by LukeTwomey** [reply]
Thanks, makes sense. Is this something that it will need access to during build time? I guess so... 

i.e. can it be a css file that its contained within the same repo (that will become available once the site has been built and served), or would it be better contained within a separate repo (one serving an api for example) so it can retrieve the css file from that, as it's already a resource that's up and running?

**Reply by LukeTwomey** [reply]
Got it working using the latter option (set up a route on a separate api to serve this css file), but curious to know if the previous approach is at all possible, by keeping the css file in the same repo?

In a completely separate API repo:

<img width="162" alt="image" src="https://user-images.githubusercontent.com/7040126/216055099-9455df7e-cff5-4228-8549-ce54f4050dc8.png">

<img width="270" alt="image" src="https://user-images.githubusercontent.com/7040126/216055132-3ef82a58-a154-46eb-b309-8efe194ede7e.png">

<img width="446" alt="image" src="https://user-images.githubusercontent.com/7040126/216055022-223ecf1e-9a23-4c2d-a6a5-d11092cefa13.png">

Back in the original repo where I'm displaying the comments:

<img width="486" alt="image" src="https://user-images.githubusercontent.com/7040126/216055348-89073fb8-9ffd-4a6f-8a92-1588bc8db37e.png">

Result:

<img width="382" alt="image" src="https://user-images.githubusercontent.com/7040126/216054878-627cd608-4c7f-42f2-8905-9ddaeac80ecd.png">

**Reply by laymonage** [reply]
You could use services such as JSDelivr's CDN that can serve a file from GitHub with the right `Content-Type` header, e.g. https://cdn.jsdelivr.net/gh/LukeTwomey/LukeTheWebDev@main/styles/blog.module.css

**Reply by LukeTwomey** [reply]
Awesome tip, thanks

**Comment by sivert-io** [comment]
![image](https://user-images.githubusercontent.com/6785315/224163018-b2d57068-6e98-4674-b81e-603109fd9d16.png)
# Still defaults to utteranc.es domain?

I haven't tested production yet, would be nice to know it works before I push it...

**Reply by laymonage** [reply]
? This discussion is for giscus, not utterances.

**Comment by EzioMercer** [comment]
How to test styles locally? I don't want to push some changes to css files somewhere to see results. Are there any options? May be there is not-frame option to have everything on my side?
